modules/core/client/controllers/core.client.controller.header.js

```diff
@@ -5,13 +5,39 @@
     .module('core')
     .controller('HeaderController', HeaderController);
 
-  HeaderController.$inject = ['menuService', '$state', '$mdSidenav'];
+  HeaderController.$inject = ['$timeout', '$mdSidenav'];
 
-  function HeaderController(menuService, $state, $mdSidenav) {
+  function HeaderController($timeout, $mdSidenav) {
     var vm = this;
 
-    vm.menus = menuService.toolbar.items;
+    vm.toggleLeft = buildDelayedToggler('left');
+    vm.toggleRight = buildDelayedToggler('right');
 
+    function debounce(func, wait, context) {
```

Comment:
Can we use lodash debounced?  Think this can be removed

https://lodash.com/docs#debounce